hardware_enablement/kmm-kernel-module-management.adoc

```diff
@@ -16,6 +16,15 @@ include::modules/kmm-installing-using-web-console.adoc[leveloffset=+2]
 include::modules/kmm-installing-using-cli.adoc[leveloffset=+2]
 include::modules/kmm-installing-older-versions.adoc[leveloffset=+2]
 
+// Added for TELCODOCS-1823
+include::modules/kmm-configuring-kmmo.adoc[leveloffset=+1]
+include::modules/kmm-unloading-kernel-module.adoc[leveloffset=+2]
+include::modules/kmm-setting-kernel-firmware-search-path.adoc[leveloffset=+2]
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about `worker.setFirmwareClassPath`, see xref:../hardware_enablement/kmm-kernel-module-management.adoc#kmm-configuring-kmmo_kernel-module-management-operator[Configuring the Kernel Module Management Operator].
```

Comment:
I'm unable to find the rule, but I think that usually we try to add a noun after resources (I know that this is a path though). So maybe consider:

```suggestion
* For more information about the `worker.setFirmwareClassPath` path, see xref:../hardware_enablement/kmm-kernel-module-management.adoc#kmm-configuring-kmmo_kernel-module-management-operator[Configuring the Kernel Module Management Operator].
```

But this might read weird with "Path` path"